controllers/pgdatabase_controller.go

```diff
@@ -265,30 +266,51 @@ func (r *PgDatabaseReconciler) handleExtensions(ctx context.Context, pgApi PgDat
 
 func (r *PgDatabaseReconciler) handleDefaultPrivileges(ctx context.Context, pgApi PgDatabaseAPI, database *apiV1.PgDatabase) error {
 	for _, schema := range database.Spec.DefaultPrivileges {
-		exists, err := pgApi.IsSchemaInDatabase(database.Name, schema.Name)
+		exists, err := pgApi.IsSchemaInDatabase(database.Name, schema.SchemaName)
 		if err != nil {
 			return err
 		}
 		if !exists {
-			if err := pgApi.CreateSchema(database.Name, schema.Name); err != nil {
+			return errors.New("Schema " + schema.SchemaName + " does not exist in database " + database.Name)
```

Comment:
If the schema is created by the admin user, it is automatically owned by the admin user.
Since we do not know who should be the owner of the schema, we can not created the schema here.